### PR TITLE
Added Jimmy Tudeski 

### PR DESCRIPTION
**Pull request by jimmy-tudeski**

### CODE REVIEW

File: registry/polkadot.json [hunk 1]
```diff
@@ -381,5 +381,13 @@
         "shortDescription": "Web3 Delegation",
         "longDescription": "The Sons of Gavin is operated by TheeWeb3Patriot, It is a web3 centered delagate that holds values of being treasury conservatives.",
         "isOrganization": false
+    },
+  {
```
Comment (stepanLav) [inline registry/polkadot.json:385]:
```suggestion
    {
```

File: registry/polkadot.json [hunk 2]
```diff
@@ -381,5 +381,13 @@
...
+        "address": "1jPw3Qo72Ahn7Ynfg8kmYNLEPvHWHhPfPNgpJfp5bkLZdrF",
+        "name": "Jimmy Tudeski - Stakenode",
+        "image": "https://github.com/jimmy-tudeski/opengov-delegate-registry/blob/master/images/Jimmy%20Profile%20.png",
```
Comment (stepanLav) [inline registry/polkadot.json:388]: use link to our repo, please
```suggestion
        "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/Jimmy%20Profile%20.png",
```

**Comment by stepanLav** [comment]
Hello @jimmy-tudeski !
Thank you for the request, could you also provide a proof of authority, by submit to system.remark from that account?

**Comment by jimmy-tudeski** [comment]
https://polkadot.subscan.io/extrinsic/21097070-2

**Comment by jimmy-tudeski** [comment]
Is there anything else that needs to be done or corrected?

**Comment by jimmy-tudeski** [comment]
Hey Stepan,

**Review by stepanLav — APPROVED** [review]

**Pull request merged**